src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,43 @@
+import cn from 'classnames';
+import { FC, useCallback, useMemo } from 'react';
+import { LoadError } from '../types/LoadError';
+import { debounce } from '../utils/debounce';
+
+interface Props {
+  loadError: LoadError,
+  setError: React.Dispatch<React.SetStateAction<LoadError>>,
+}
+
+export const ErrorNotification:FC<Props> = ({ loadError, setError }) => {
+  const disableError = useCallback(() => {
+    setError((current) => ({
+      ...current,
+      status: false,
+    }));
+  }, [setError]);
+
+  const debouncedDisableError = useMemo(() => (
+    debounce(disableError, 3000)
+  ), [disableError]);
+
+  if (loadError.status) {
+    debouncedDisableError();
+  }
```

Comment:
You can realize it by `useEffect`.
Debounce intended for slightly different tasks. 
Create `useEffect`  which will create timeOut when an error update and will unmount this timer after 3 seconds

Reply:
I support this remark, it is important to make similar parts using `useEffect`.